src/main/java/uk/gov/pay/ledger/payout/state/PayoutState.java

```diff
@@ -18,18 +18,18 @@
     PAID_OUT("paidout", true),
     FAILED("failed", true);
 
-    private final String status;
+    private final String state;
```

Comment:
I believe this should still be `status`, the current public API models looks like: 
```
{
  "state": {
    "status": "the_status"
  }
}
```